### PR TITLE
Remove trailing spaces/tabs

### PR DESCRIPTION
**Pull request by rbuj**
```
find . \( -name '*.h' -o -name '*.c' \) -exec sed -i 's/[[:space:]]*$//' {} \;
find . \( -name '*.h' -o -name '*.c' \) -exec sed -i 's/\t*$//' {} \;
```

### CODE REVIEW

File: src/core/keybindings.c [hunk 1]
```diff
@@ -2846,8 +2846,8 @@ process_workspace_switch_grab (MetaDisplay *display,
                                                   window,
                                                   event->xkey.time);
             }
-            
-          
+
```
Comment (raveit65) [inline src/core/keybindings.c:2849]: I think you can remove the double line too, when you already touch this file, or not?

File: src/core/keybindings.c [hunk 2]
```diff
@@ -3617,39 +3617,39 @@ handle_rename_workspace_callback(GIOChannel *ioc, GIOCondition cond, gpointer da
...
   if (cond & G_IO_HUP)
     {
       meta_topic (META_DEBUG_KEYBINDINGS, "handle_rename_workspace_callback: done.\n");
       g_free(workspace_index);
       already_displaying_rename_workspace = FALSE;
       return FALSE;
     }
-  
-  
+
```
Comment (raveit65) [inline src/core/keybindings.c:3634]: Same here

File: src/core/keybindings.c [hunk 3]
```diff
@@ -3658,23 +3658,23 @@ handle_rename_workspace_callback(GIOChannel *ioc, GIOCondition cond, gpointer da
...
```
Comment (raveit65) [inline src/core/keybindings.c:3661]: Same here

File: src/core/util.c [hunk 4]
```diff
@@ -658,27 +658,27 @@ meta_show_entry_dialog(const char *message,
...
+
   if (cancel_text)
     {
       argvl[i++] = "--cancel-label";
       argvl[i++] = cancel_text;
     }
   argvl[i] = NULL;
-  
-  
+
```
Comment (raveit65) [inline src/core/util.c:680]: Same here

File: src/core/window.c [hunk 5]
```diff
@@ -4997,8 +4997,8 @@ meta_window_move_resize_request (MetaWindow *window,
                                       y,
                                       width,
                                       height);
-  
-   }    
+
```
Comment (raveit65) [inline src/core/window.c:5000]: Do we need this extra line?

**Comment by raveit65** [comment]
Rest looks OK for me, hopefully this doesn't create conflicts with actual open PRs.

**Comment by rbuj** [comment]
@raveit65 done

**Pull request merged**